src/app/infobox/infobox.component.ts

```diff
@@ -41,8 +49,14 @@ export class InfoboxComponent implements OnInit {
         }
 
     });
+  }
 
+  if (synthesisEnable = true) {
```

Comment:
@harshit98 while you are coding something in javascript, remember that it is an asynchronous language.
where is the correct block in the whole code which runs immediately after receiving results?
it the block between 39 to 51 that is the block inside `this.response$.subscribe`